pkg/serverless/proxy/transport.go

```diff
@@ -14,13 +14,16 @@ import (
 	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/serverless/invocationlifecycle"
+	serverlessLog "github.com/DataDog/datadog-agent/pkg/serverless/logs"
```

Comment:
here I definitely agree that naming the import is needed as it can conflict with the next line